content/en/technical-overview/architecture/_index.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
nitpick: In the Hextra theme, we attempted to keep the section index pages lightweight by using `cards` or the `subPages` shortcode. I suggest separating this whole page into an `Overview`, and keeping the `_index.md` for the summary of the section.

Reply:
Ok, I changed this but am unsure this is the requested change. The _index page looks kinda empty now.